apps/src/templates/StudentsBeyondHoc.jsx

```diff
@@ -6,7 +6,14 @@ import CourseBlocksStudentGradeBands from './studioHomepages/CourseBlocksStudent
 import VerticalImageResourceCardRow from './VerticalImageResourceCardRow';
 import { LocalClassActionBlock } from './studioHomepages/TwoColumnActionBlock';
 import { tutorialTypes } from './tutorialTypes.js';
-import { pre2017MinecraftCards } from './congratsBeyondHocActivityCards';
+import {
+  pre2017MinecraftCards,
+  // newMinecraftCards,
```

Comment:
one rule of thumb is to always have a comment explaining why code is commented-out, though it's better to never commit commented-out code.